components/home/sections/contact.js

```diff
@@ -1,98 +1,118 @@
-                className="block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300"
+                htmlFor='email'
+                className='block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300'
               >
                 Email
               </label>
               <input
-                type="email"
-                id="email"
-                className="shadow-sm bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-primary-500 focus:border-primary-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-primary-500 dark:focus:border-primary-500 dark:shadow-sm-light"
+                type='email'
+                id='email'
+                className='shadow-sm bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-primary-500 focus:border-primary-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-primary-500 dark:focus:border-primary-500 dark:shadow-sm-light'
                 required
               />
             </div>
-            <div className="sm:col-span-2">
+            <div className='sm:col-span-2'>
               <label
-                htmlFor="message"
-                className="block mb-2 text-sm font-medium text-gray-900 dark:text-gray-400"
+                htmlFor='message'
+                className='block mb-2 text-sm font-medium text-gray-900 dark:text-gray-400'
               >
                 Message
               </label>
               <textarea
-                id="message"
-                rows="6"
-                className="block p-2.5 w-full text-sm text-gray-900 bg-gray-50 rounded-lg shadow-sm border border-gray-300 focus:ring-primary-500 focus:border-primary-500 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-primary-500 dark:focus:border-primary-500"
-                placeholder="Leave your comment..."
+                id='message'
+                rows='6'
+                className='block p-2.5 w-full text-sm text-gray-900 bg-gray-50 rounded-lg shadow-sm border border-gray-300 focus:ring-primary-500 focus:border-primary-500 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-primary-500 dark:focus:border-primary-500'
+                placeholder='Leave your comment...'
               ></textarea>
             </div>
             <button
-              type="submit"
-              className="hover:bg-pyconug-lightBlue border-2 hover:text-white border-pyconug-lightBlue py-2 px-10 m-2  rounded-3xl"
+              type='submit'
+              className='hover:bg-pyconug-lightBlue border-2 hover:text-white border-pyconug-lightBlue py-2 px-10 m-2  rounded-3xl'
             >
               Send message
             </button>
           </form>
         </div>
       </div>
+      <ToastContainer />
```

Comment:
we could refactor to have the provider instantiated in the `root layout` than calling it on every use case

Reply:
hi @mosespace sorry i think you got me wrong...by `we could refactor to have the provider instantiated in the root layout than calling it on every use case` i didnt mean making the `root layout` a `client component`.....
i meant we could have a provider.... that is called in the root layout, the provider can be a client component..... server components can render client components...
so all notifications configs can be applied through a provider globally....